PiView_AG/Utils.py

```diff
@@ -14,11 +14,8 @@
 
 
 class Utils:
-
-    def __init__(self):
-        return None
```

Comment:
This is not needed and in fact, returning anything from the `__init__` method is not allowed.

Reply:
:-D I stand corrected